chalice/app.py

```diff
@@ -426,11 +466,14 @@ def __call__(self, event, context):
         view_function = route_entry.view_function
         function_args = [event['pathParameters'][name]
                          for name in route_entry.view_args]
+        body = event['body']
+        if event.get('isBase64Encoded'):
+            body = base64decode(body)
```

Comment:
Given the request object handles json loading, seems reasonable to move this decoding responsibility into the `Request` class as well.